api/src/main/java/com/nicknathanjustin/streamercontracts/donations/DonationsApiController.java

```diff
@@ -53,6 +59,16 @@ public ResponseEntity createDonation(@RequestBody @NonNull final CreateDonationR
 
         final ContractModel contract = contractService.createContract(proposer, streamer, game, createDonationRequest.getBounty());
         donationService.createDonation(contract, proposer, createDonationRequest.getAmount(), contract.getProposedAt(), paypalPaymentId, authorizationId);
+
+        // send notification to FE
+        final String title = "New bounty from " + contract.getProposer().getTwitchUsername();
```

Comment:
Nit: Make "New bounty from %s" a constant static field and reference in both controllers